content/en/blog/2019/multi-cluster-mesh-automation/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Multi-mesh configuration and service discovery using Admiral
+subtitle: Configuration automation for Istio multi-mesh.
+description: Automating Istio configuration for Istio deployments (clusters) that work as a single mesh.
+publishdate: 2019-12-28
+attribution: Anil Attuluri (Intuit), Jason Webb (Intuit)
+keywords: [traffic-management,automation,configuration,multicluster,multi-mesh,gateway,federated,globalidentifer]
+target_release: 1.5
+---
+
+When we read the blog post [Multi-Mesh Deployments for Isolation and Boundary Protection](https://istio.io/blog/2019/isolated-clusters/) we could immediately relate to some of the problems mentioned and how we are using [Admiral](https://github.com/istio-ecosystem/admiral), an open source project under istio-ecosystem to solve these.
+
+## Background
+
+We were off to a good start with Istio but quickly realized the configuration for multi-cluster was complicated and would be challenging to maintain over time. We picked an opinionated version of [Multi-Cluster Istio Service Mesh with local control planes](https://istio.io/docs/setup/install/multicluster/gateways/#deploy-the-istio-control-plane-in-each-cluster) for scalability and other operational reasons. Having picked this model, we had the following  key requirements to solve for before allowing wide adoption of Istio Service Mesh:
+- Creation of service DNS decoupled from the namespace (Uniform service naming mentioned [here](https://istio.io/blog/2019/isolated-clusters/#features-of-multi-mesh-deployments))
+- Service Discovery across many clusters
+- Support Active-Active & HA/DR deployments with services being deployed in globally unique namespaces in clusters in different regions.
```

Comment:
Active-Active Support with HA/DR deployments. We also needed to support these critical resiliency patterns with services being deployed in globally unique namespaces across discrete clusters.